### ISSUE TITLE
ELSE path not installed correctly

### ISSUE THREAD

**Issue by H-MLim** [plugdata-team/plugdata#1423]
related with #1334, I found that ELSE externals have been not installed in its path correctly since 0.8.4 builds. it showed me the weird folder icon with arrows. when I click those folders, they output the error messages like this:

> C:\users\\(username)\Documents\plugdata\Abstractions(Documentation) is unavaliable.  If the location is on this PC, make sure the device or drive is connected or the disc is inserted, and then try again. If the location is on a network, make sure you're connected to the network or internet, and then try again. If the location still can't be found, it might have been moved or deleted.

by the way there are no any files in the folders of `Externals` and `Extra` except `Extra\GS`
(win11)

![image](https://github.com/plugdata-team/plugdata/assets/68152553/8a665419-9db1-4d3c-9d0b-7d63b4c9d607)

![image](https://github.com/plugdata-team/plugdata/assets/68152553/ba52254d-2e2f-4242-8e02-35e729508a8d)

![image](https://github.com/plugdata-team/plugdata/assets/68152553/e6608a88-6e26-4969-9a97-21c317ba4290)

**Comment by dromer** [comment]
These directories are simlinks that should resolve to the specific Version directory.

**Comment by H-MLim** [comment]
Ok..so you mean this is a bug?

**Comment by timothyschoen** [comment]
hmm this is strange, but if you just delete the "plugdata" folder from settings and restart plugdata, it will probably fix it

**Comment by H-MLim** [comment]
hmm. I did it once. but still same error messages.. (but also, 0.8.3v hadn't any error messages and I could access every folder)

**Comment by H-MLim** [comment]
![image](https://github.com/plugdata-team/plugdata/assets/68152553/7b391224-b26f-4f2f-aa20-7397e1319a99)
that's strange. it seems it recognizes ELSE. but it can't load.

**Comment by H-MLim** [comment]
![image](https://github.com/plugdata-team/plugdata/assets/68152553/a72465dd-eaf9-444d-ab7f-6a5640ba146e)

it still can load other externals like cyclone or Gem, so it seems it's not the whole directory problem.

**Comment by H-MLim** [comment]
![image](https://github.com/plugdata-team/plugdata/assets/68152553/ef6866d3-a6bb-485e-b541-491955b91566)

but the interesting thing is...it can load some ELSE externals like `note`!

**Comment by H-MLim** [comment]
this fixed on build 6867272

**Comment by timothyschoen** [comment]
I don't think this is fixed: whenever I change something in plugdata's files, it will unpack the resources again. So looks this like process sometimes works and sometimes fails for you.
